docs/docs-content/release-notes/known-issues.md

```diff
@@ -0,0 +1,40 @@
+---
+
+Throughout the development lifecycle of Palette, known issues may arise that affect the user experience. Use this page
+to review and stay informed about the status of known issues in Palette. As issues are resolved, this page is updated.
+
+## Active Known Issues
+
+The following table lists all known issues that are currently active and affecting users.
+
+| Description                                                                                                                                                                                                                                                                                                                                        | Workaround                                                                                                                                                                                                                                                                     | Publish Date   | Product Component |
+| -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------- | ----------------- |
+| Conducting cluster node scaling operations on a cluster undergoing a backup can lead to issues and potential unresponsiveness.                                                                                                                                                                                                                     | To avoid this, ensure no backup operations are in progress before scaling nodes or performing other cluster operations that change the cluster state                                                                                                                           | April 14, 2024 | Clusters          |
+| Palette automatically creates a security group for worker nodes using the format `<cluster-name>-node`. If a security group with the same name already exists in the VPC, the cluster creation process fails.                                                                                                                                      | To avoid this, ensure that no security group with the same name exists in the VPC before creating a cluster.                                                                                                                                                                   | April 14, 2024 | Clusters          |
```

Comment:
```suggestion
| Palette automatically creates an AWS security group for worker nodes using the format `<cluster-name>-node`. If a security group with the same name already exists in the VPC, the cluster creation process fails.                                                                                                                                      | To avoid this, ensure that no security group with the same name exists in the VPC before creating a cluster.                                                                                                                                                                   | April 14, 2024 | Clusters          |
```